Meta/check-markdown.sh

```diff
@@ -23,4 +23,4 @@ if [ -z "$SERENITY_SOURCE_DIR" ] ; then
     export SERENITY_SOURCE_DIR
 fi
 
-find AK Base Documentation Kernel Meta Ports Tests Userland -path 'Ports/*/*' -prune -o -type f -name '*.md' -print0 | xargs -0 "${MARKDOWN_CHECK_BINARY}" README.md
+find AK Base Documentation Kernel Meta Ports Tests Userland -path 'Ports/*/*' -prune -o -type f -name '*.md' -print0 | xargs -0 "${MARKDOWN_CHECK_BINARY}" README.md CONTRIBUTING.md
```

Comment:
Commit message mentions FAQ; where is it? :)

Reply:
I decided that moving FAQ isn't worth it when we're not getting rid of Documentation, and this originally added markdown checking to FAQ as well. However, since we're not currently checking anything in Documentation I wanted to leave that up to another PR if anyone wants it.

Reply:
But yes, commit message needs adjustment.